<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskRevolutionParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskRevolutionParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelAxis">
       <property name="text">
        <string>Axis:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="axis">
       <item>
        <property name="text">
         <string>Base X axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base Y axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base Z axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Horizontal sketch axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vertical sketch axis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Select reference...</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Angle:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Gui::QuantitySpinBox" name="revolveAngle" native="true">
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
       <property name="unit" stdset="0">
        <string notr="true">deg</string>
       </property>
       <property name="minimum" stdset="0">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum" stdset="0">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep" stdset="0">
        <double>10.000000000000000</double>
       </property>
       <property name="value" stdset="0">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxReversed">
       <property name="text">
        <string>Reversed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMidplane">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Symmetric to plane</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
